iop/sio/mx4sio_bd/src/sio2man_hook.c

```diff
@@ -97,26 +97,26 @@ static void _sio2man_unhook(iop_library_t *lib)
         return;
     }
 
-    ioplib_hookExportEntry(lib, 23, _23_psio2_pad_transfer_init);
-    ioplib_hookExportEntry(lib, 24, _24_psio2_mc_transfer_init);
-    ioplib_hookExportEntry(lib, 25, _25_psio2_transfer);
-    ioplib_hookExportEntry(lib, 26, _26_psio2_transfer_reset);
-    ioplib_hookExportEntry(lib, 46, _46_psio2_pad_transfer_init);
-    ioplib_hookExportEntry(lib, 47, _47_psio2_mc_transfer_init);
-    ioplib_hookExportEntry(lib, 48, _48_psio2_mtap_transfer_init);
+    modhookk_hookExportEntry(lib, 23, _23_psio2_pad_transfer_init);
```

Comment:
How did the CI build this succesfully? `modhookk` with double 'k'?

Reply:
Lol, didn't saw that. Will recheck this later!